test/ERC20VotesPartialDelegationUpgradeable.t.sol

```diff
@@ -317,6 +325,98 @@ contract Delegate is PartialDelegationTest {
     tokenProxy.delegate(delegations);
     vm.stopPrank();
   }
+
+  function testFuzz_RevertIf_PartialDelegationLimitExceeded(
+    address _actor,
+    uint256 _amount,
+    uint256 _numOfDelegatees,
+    uint256 _seed
+  ) public {
+    vm.assume(_actor != address(0));
+    _amount = bound(_amount, 0, type(uint208).max);
+    _numOfDelegatees =
+      bound(_numOfDelegatees, tokenProxy.MAX_PARTIAL_DELEGATIONS() + 1, tokenProxy.MAX_PARTIAL_DELEGATIONS() + 500);
+    PartialDelegation[] memory delegations = _createValidPartialDelegation(_numOfDelegatees, _seed);
+    vm.startPrank(_actor);
+    tokenProxy.mint(_amount);
+    vm.expectRevert(
+      abi.encodeWithSelector(
+        PartialDelegationLimitExceeded.selector, _numOfDelegatees, tokenProxy.MAX_PARTIAL_DELEGATIONS()
+      )
+    );
+    tokenProxy.delegate(delegations);
+    vm.stopPrank();
+  }
+
+  function testFuzz_RevertIf_DuplicateOrUnsortedDelegatees(
+    address _actor,
+    uint256 _amount,
+    uint256 _numOfDelegatees,
+    address _newDelegatee,
+    uint256 _seed
+  ) public {
+    vm.assume(_actor != address(0));
+    _amount = bound(_amount, 0, type(uint208).max);
+    _numOfDelegatees = bound(_numOfDelegatees, 2, tokenProxy.MAX_PARTIAL_DELEGATIONS());
+    PartialDelegation[] memory delegations = _createValidPartialDelegation(_numOfDelegatees, _seed);
+    address lastDelegatee = delegations[delegations.length - 1]._delegatee;
+    vm.assume(_newDelegatee <= lastDelegatee);
```

Comment:
```suggestion
    vm.assume(_replacedDelegatee <= lastDelegatee);
```

Reply:
renamed in https://github.com/voteagora/ERC20VotesPartialDelegationUpgradeable/pull/40/commits/ad8a10b1ae70d05dc3391b1ffce8c6d5be7894ff